views/form.html

```diff
@@ -1,9 +1,9 @@
 <!--
-Form for OnBoarding process
-@author Laxmi Kandel
--->
+   Form for OnBoarding process
+   @author Laxmi Kandel
+   -->
 <!doctype html>
-<html lang="en">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
```

Comment:
I don't know what the xmlns="http://www.w3.org/1999/html" means but i'd guess you don't need it